pkgs/applications/misc/qt-box-editor/default.nix

```diff
@@ -0,0 +1,37 @@
+{ stdenv
+, fetchFromGitHub
+, qtbase
+, qtsvg
+, qmake
+, leptonica
+, tesseract
+}:
+
+stdenv.mkDerivation rec {
+  pname = "qt-box-editor";
+  version = "unstable-2019-07-12";
+
+  src = fetchFromGitHub {
+    owner  = "zdenop";
```

Comment:
Please don't vertically align these. This style isn't maintained in nixpkgs

Reply:
Fixed.